src/pages/docs/octopus-rest-api/octopus.server.exe-command-line/builtin-worker.md

```diff
@@ -30,7 +30,7 @@ Where [<options>] is any of:
                              The username for a user account the built-in
                                worker should use when executing custom scripts,
                                like 'octopus-worker' or 'octopus-
-                               worker@mycomany.com'. Default is to use the same
+                               worker@mycompany.com'. Default is to use the same
```

Comment:
This is probably based on an actual program's buggy output, so please consider this as a bug report.